mercury-operator/src/test/java/com/redhat/mercury/operator/controller/ServiceDomainControllerTest.java

```diff
@@ -74,7 +65,7 @@ public void afterEach() {
         deleteDeployment();
         deleteService();
         deleteKafkaTopic();
-        deleteIntegrationConfigMap();
+        deleteDirectConfigMap();
```

Comment:
I don't see you validate the configmaps are properly created or recreated when deleted